cli.js

```diff
@@ -7,9 +7,31 @@ const isCheckFlag = (argument) => argument === '--check' || argument === '-c'
 const isHelpFlag = (argument) => argument === '--help' || argument === '-h'
 const isVersionFlag = (argument) =>
   argument === '--version' || argument === '-v'
+const isQuietFlag = (argument) => argument === '--quiet' || argument === '-q'
 
 const cliArguments = process.argv.slice(2)
 const isCheck = cliArguments.some(isCheckFlag)
+const isQuiet = cliArguments.some(isQuietFlag)
+
+function stdout(outputIfTTY = '', alwaysOutput = outputIfTTY) {
+  if (isQuiet) return
+  const isTerminal = !!process.stdout.isTTY
+  if (isTerminal) {
+    console.log(outputIfTTY)
+  } else if (alwaysOutput !== null) {
+    console.log(alwaysOutput)
+  }
+}
```

Comment:
I can't understand why are we checking `process.stdout.isTTY` and what's the purpose of the `alwaysOutput` parameter.

Isn't this enough?

```js
const stdout = isQuiet ? () = {} : console.log
```

Reply:
You could do that for absolute output, I was planning for further changes, but I'm now going to move those to a new PR. I will revert this and stderr to a simpler form.